frontend/src/app/app.module.ts

```diff
@@ -1,7 +1,12 @@
 import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 
-import { AppRoutingModule } from './app-routing.module';
```

Comment:
Any reason why we are deleting the AppRoutingModule?

Reply:
Yea it was not needed now
 we will see  it after I work on the navigation.